File: tests/indexing/parsing/lang/fixtures/c/Formatting.c
int allman(int a, int b)
{
    if (a > b)
    {
        return a;
    }
    return b;
}

int multi_line_signature(
    int first,
    int second
) {
    return first + second;
}

int multi_line_header(int a, int b, int c) {
    int total = 0;
    while (a > 0
        && b > 0
        && c > 0) {
        total += a;
        a -= 1;
    }
    return total;
}

File: tests/indexing/parsing/lang/fixtures/ruby/Formatting.rb
class Styles
  def combine(
    first,
    second
  )
    first * second + first
  end

  def label(code, count)
    case code
    when 1 then "one:#{count}"
    when 2 then "two:#{count * 2}"
    else "many:#{count}"
    end
  end
end
